transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_level_wrappers.py

```diff
@@ -19,17 +19,23 @@
 SORT_SEMANTIC_NORMALISED = "SORT_SEMANTIC_NORMALISED"
 
 
-def semantic_sort(df, logger, title_column_name):
-    return sort_sem(files_df=df, logger=logger, title_column_name=title_column_name)
+def semantic_sort(df, logger, title_column_name, language_column_name):
+    return sort_sem(
+        files_df=df, logger=logger, title_column_name=title_column_name, language_column_name=language_column_name
+    )
 
 
-def semantic_sort_normalised(df, logger, title_column_name):
+def semantic_sort_normalised(df, logger, title_column_name, language_column_name):
     check_and_update_title(df)
-    return sort_sem(df, logger, title_column_name)
+    return sort_sem(df, logger, title_column_name, language_column_name)
```

Comment:
here positional arguments notation used and at line no. 24 used keyword arguments notation for the same function invocation. can we have one invocation convention across the code. or is there any specific reason used different convention

Reply:
corrected.